pkg/comp-functions/functions/common/non_SLA_prom_rules.go

```diff
@@ -0,0 +1,162 @@
+package common
```

Comment:
please no CAPITAL letters in filenames, that can get messy.

Reply:
fixed